frontend/src/reducers/accessRequestReducer.tsx

```diff
@@ -19,6 +28,13 @@ const accessRequestReducer = (state = initialState, action: IStoreAccessRequests
           items: [...action.pagedAccessRequests.items],
         },
       };
+    case actionTypes.STORE_ACCESS_REQUEST:
+      return {
+        ...state,
+        accessRequest: {
+          ...action.accessRequest,
```

Comment:
you may need to explicitly copy all levels of nested data (ie, what I did with the array above).
https://redux.js.org/recipes/structuring-reducers/immutable-update-patterns

or use immer, which is used in the slices:
https://github.com/immerjs/immer